src/frontend/packages/store/src/reducers/current-user-roles-reducer/recently-visited.reducer.helpers.ts

```diff
@@ -62,3 +63,11 @@ export function cleanRecentsList(state: IRecentlyVisitedState, endpointGuids: st
   // Convert the array back into a map
   return filtered.reduce(recentArrayToMap, {});
 }
+
+export function clearEntityFromRecentsList(state: IRecentlyVisitedState, action: EntityDeleteCompleteAction): IRecentlyVisitedState {
+  // Remove entity from the map if it exists
+  const fav = action.asFavorite();
```

Comment:
Haven't debugged but this isn't working/isn't being hit. I deleted an application via the delete app stepper but the application remained in the favourites list in the home page and the recent's list. I can see the `[Entity] Entity delete complete` action for the app and route entities were dispatched ok.  Interestingly, even after a refresh, they remained in the favourites list even though the http requests were 200s.

Reply:
Fixed - the type and id were transposed in the action, so the guid generated for the favorite when deleting was incorrect.